packages/legend-lego/src/application/DocumentationLink.tsx

```diff
@@ -70,14 +70,16 @@ export const DocumentationPreview: React.FC<{
     applicationStore.documentationService.getDocEntry(documentationKey);
   const textContent = text ?? documentationEntry?.text;
 
-  if (!documentationEntry || !textContent) {
+  if (!textContent) {
```

Comment:
I think we should revert this, the intended behavior for `DocumentationLink` is if the entry is not available, we will not shot the hint

Reply:
my bad, I misread 🙇